Backpack/src/main/res/drawable/bpk_bottom_nav_background.xml

```diff
@@ -1,10 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-  <item android:drawable="@color/bpkBackground" />
```

Comment:
these were all not necessary as we've got a separate night mode nav to deal with the divider and all these colours were white in light mode